airflow/utils/helpers.py

```diff
@@ -147,6 +148,68 @@ def chain(*tasks):
         up_task.set_downstream(down_task)
 
 
+def popen_and_tail(cmd, shell=False, bufsize=512, env=None, verbose=True):
```

Comment:
There is no good reason to expose `shell` on *nix. Let's just remove it from the API.